data/tags.json

```diff
@@ -113,6 +113,8 @@
       "<https://dontasktoask.com/>",
       "- Please don’t just say “hello”",
       "<https://no-hello.com/>",
+      "- How To Ask Questions The Smart Way",
+      "<http://catb.org/~esr/faqs/smart-questions.html>",
```

Comment:
This page is almost 12,000 words. Do you really think anyone who needs it will use it? :)

Which is to say: I do not think we should include it, but I'm interested in whether this is just A Thing that fits the list or if you think it's an important addition in some way.

Reply:
i feel the other links dont cover every important topic, are very specific and already have their own tags (nohello, ask, rd)

if you do decide to keep it, i think it would be better to replace other links with this one
if the length of the post is an issue i can abbreviate it

Reply:
i just noticed this is already included in `problemsolving` tag. it makes more sense under `goodpractices`

Reply:
The length of the post is absolutely an issue. :) The purpose of these tags is to save time saying the same things that have be said every other day, and to actually communicate successfully to people who are not communicating well. Even if I had no issues with ESR's content, the length makes it useless. The only people who will read all that are those who think themselves above the problems it describes, marvelling at their own superiority.

Please remove, and I'll strike it from the other tag in another commit, too.